terminology.md

```diff
@@ -496,12 +496,14 @@ Some real-world examples:
 
 The idea that when differing perspectives come into conflict, our decisions ought to prioritize them in the following (descending) order:
 
-1. End users
+1. End users (tool and website users)
```

Comment:
We can even just replace instead of adding here if you prefer; I was debating about the right-sized wording myself.

Reply:
It feels hard to elaborate the full set of End users so, for sake of caution, let's keep the collective term for now.